### PR TITLE
8263593: Fix multiple typos in hsdis README

### PR DESCRIPTION
**Pull request by henri-tremblay**
---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8263593](https://bugs.openjdk.java.net/browse/JDK-8263593): Fix multiple typos in hsdis README

### Reviewers
 * [Tobias Hartmann](https://openjdk.java.net/census#thartmann) (@TobiHartmann - **Reviewer**)

### Download
To checkout this PR locally:
`$ git fetch https://git.openjdk.java.net/jdk pull/2685/head:pull/2685`
`$ git checkout pull/2685`

To update a local copy of the PR:
`$ git checkout pull/2685`
`$ git pull https://git.openjdk.java.net/jdk pull/2685/head`

### CODE REVIEW

**Comment by henri-tremblay** [comment]
I'm already on https://www.oracle.com/technical-resources/oracle-contributor-agreement.html. How can I be recognized on github?

File: src/utils/hsdis/README [hunk 1]
```diff
@@ -70,8 +70,8 @@ other configurations that don't include the full NLS support.
 The makefile looks for the sources in build/binutils or you can
 specify it's location to the makefile using BINUTILS=path.  It will
```
Comment (hseigel) [inline src/utils/hsdis/README:71]: This should be "its", not "it's"

File: src/utils/hsdis/README [hunk 2]
```diff
@@ -101,9 +101,9 @@ http://sourceforge.net/projects/mingw-w64.  Grab a copy of the
 complete toolchain and unpack it somewhere.  Put the bin directory of
 the toolchain in your path.  The mingw installs contain cross compile
 versions of gcc that are named with a prefix to indicate what they are
-targetting and you must tell the Makefile which one to use.  This
+targeting and you must tell the Makefile which one to use.  This
 should either be i586-mingw32msvc or x86_64-pc-mingw32 depending on
-which on you are targetting and there should be a version of gcc in
+which on you are targeting and there should be a version of gcc in
```
Comment (hseigel) [inline src/utils/hsdis/README:106]: "which one' instead of 'which on' ?

Comment (henri-tremblay) [reply src/utils/hsdis/README:106]: Indeed

**Comment by henri-tremblay** [comment]
I've fixed the new typos found by @hseigel

**Comment by henri-tremblay** [comment]
@hseigel Why is my commit message not accepted? Format looks good to me. What am I missing?

**Comment by TobiHartmann** [comment]
You need to link the PR to this bug [JDK-8263593](https://bugs.openjdk.java.net/browse/JDK-8263593) (see [/issue](https://wiki.openjdk.java.net/display/SKARA/Pull+Request+Commands#PullRequestCommands-/issue) command) and update the title accordingly.

**Review by TobiHartmann — APPROVED** [review]
Looks good to me.

**Pull request closed without merging**